docs/v2/token_filters/ascii_folding.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: ASCII Folding
+---
+
+The ASCII folding filter strips away diacritical marks (accents, umlauts, tildes, etc.) while leaving the base character intact.
+It is supported for all tokenizers besides the [exact](/v2/tokenizers/available_tokenizers/exact) tokenizer.
+
+To enable, append `ascii_folding=true` to the tokenizer's arguments.
+
+```sql
+CREATE INDEX search_idx ON mock_items
+USING bm25 (id, (description::pdb.simple('ascii_folding=true')))
+WITH (key_field='id');
+```
+
+To demonstrate this token filter, let's compare the output of the following two statements:
+
+```sql
+SELECT
+  'Café naïve coöperate'::pdb.simple::text[],
```

Comment:
@eeeebbbbrrrr, @rebasedming : Might it be worth having a separate namespace for tokenizers, to make them more discoverable?
```sql
  'Café naïve coöperate'::tokenizer.simple::text[],
  'Café naïve coöperate'::tokenizer.icu::text[],
```